internal/api/grpc/system/quota_integration_test.go

```diff
@@ -28,7 +28,7 @@ func TestServer_QuotaNotification_Limit(t *testing.T) {
 	percent := 50
 	percentAmount := amount * percent / 100
 
-	_, err := Tester.Client.System.AddQuota(SystemCTX, &system.AddQuotaRequest{
```

Comment:
Test for the depricated AddQuota are not necessary anymore?

Reply:
Not as integration test, IMO